utilities/zookeeper/R/corrector-aardvark.R

```diff
@@ -44,6 +44,52 @@ default_state_params <- function(
   )
 }
 
+# 
+special_flags_marker <- function(
+  geo_value = NA,
+  signal = NA,
+  time_value = NA,
+  max_lag = NA) {
+  tibble::tibble(
+    geo_value = geo_value,
+    signal = signal,
+    time_value = time_value,
+    max_lag = max_lag
+  )
+}
+
+#' @examples
+#' special_flags <- special_flags_marker(c('wa','oh'), 
+#'                                      c('deaths_incidence_num','deaths_incidence_num'),
+#'                                      list(lubridate::ymd(c("2020-12-16", "2020-12-17","2020-12-23", "2020-12-24","2020-12-29")),
+#'                                            seq(lubridate::ymd("2021-02-20"), lubridate::ymd(time_value_flag_date),by=1)),
+#'                                      c(14,60))
+#'    
+#'             
+#'  Two helper functions
+#'  (1) Add special flagging columns, this function will create columns like "flag_bad_va", "flag_bad_wa", etc.    
+special_flagging <- function(df,special_flags){
+  for (i in 1:nrow(special_flags)){
+    varname <- paste0('flag_bad_',special_flags$geo_value[i])
+    df <- mutate(df,{{varname}} := (.data$geo_value == special_flags$geo_value[i] &
```

Comment:
`:=` is in the `rlang` package right? We need to import it somewhere. Easiest is to add `@importFrom rlang ':='` in the roxygen.

Reply:
(replace apostrophe with the tick above the tab key)